app/models/participant_profile/npq.rb

```diff
@@ -6,10 +6,21 @@ class ParticipantProfile::NPQ < ParticipantProfile
 
   has_one :validation_data, class_name: "NPQValidationData", foreign_key: :id, dependent: :destroy
 
+  self.validation_steps = %i[identity decision]
```

Comment:
`.freeze` ?

Reply:
Ah, good idea! One note though - in not so distant future those steps will need to take a few additional options like `on` to specify that given step decisions are to be stored against another object. On one hand it could live on that other record, but I think I'd prefer the whole validation logic to be encapsulated in a single place.